region/lowernorfair/west.json

```diff
@@ -589,7 +589,17 @@
             "runways": [
               {
                 "length": 12,
-                "requires": null
+                "requires": [
+                  {"or": [
+                    "heatProof",
+                    "canHeatRun"
+                  ]},
+                  "reset",
```

Comment:
I think this needs to be thought out a bit. I think it could be an array of all nodes you could use to properly reset. That way, we'll be able to use the same thing at Early Supers for travelling from node 1 to 2.

Reply:
{"reset": [1,2,3]} ?

Reply:
something like that. Maybe `resetFromNode`?

Reply:
It looks like we're going to leave this for a later PR with issue #121, so resolving conversation.